docs/workflows/README.md

```diff
@@ -0,0 +1,184 @@
+<p style="text-align: center">
+  <a href="https://commercetools.com/">
+    <img alt="commercetools logo" src="https://unpkg.com/@commercetools-frontend/assets/logos/commercetools_primary-logo_horizontal_RGB.png">
+  </a><br/>
+    <a href="https://www.paypal.com/de/business/accept-payments">
+    <img alt="PayPal logo" src="https://www.paypalobjects.com/webstatic/de_DE/i/de-pp-logo-200px.png">
+  </a><br>
+</p>
+
+In this folder you can find simplified example workflows for the PayPal commercetools connector, which should be sufficient to reproduce demonstrated payments. For all details please refer to [Architecture.pdf](../Architecture.pdf). 
+
+Here we summarize the actual workflows for:
+1. using only the connector for the backend side implementation 
+2. using the [PayPal client](https://www.npmjs.com/package/paypal-commercetools-client) in commercetools Frontend (CoFe) [integration](https://github.com/mediaopt/paypal-commercetools-cofe-integration/tree/main).
+
+The workflow that only includes backend calls through Postman is also provided as a [workflow Postman Collection](PayPal-commercetools-workflow.postman_collection.json).
```

Comment:
Here and in the next sentence you use `also`. So I think here it can just be removed to be a bit prettier to read.